src/APICaller.js

```diff
@@ -0,0 +1,21 @@
+async function fetchWeatherData(url) {
+  const response = await fetch(url, {
+    mode: 'cors',
+  });
+  const responseToJson = await response.json();
+
+  return responseToJson;
+}
+
+function getUrl(place) {
+  const apiKey = '9a40fda64112122250ff2786a779a184';
+  const url = `http://api.openweathermap.org/data/2.5/weather?q=${place}&APPID=${apiKey}&units=metric`;
```

Comment:
try to encode URI components, since if you'll input place with space char or some other special char it might break for you.